leasing/report/invoice/invoicing_review.py

```diff
@@ -336,6 +340,61 @@ def get_incorrect_rent_shares_data(self, service_unit_ids, cursor):
 
         return data
 
+    def get_ongoing_rent_without_rent_shares_data(self, service_unit_ids, cursor):
+        today = datetime.date.today()
+
+        query = """
+            SELECT li.identifier as lease_id,
```

Comment:
naming the identifier as id can be confusing, since they are generally two separate things. How about using `as lease_identifier` instead?

Reply:
Should I change lease_id in report to lease_identifier as well or can I leave it as lease_id?

Reply:
Actually the report just shows "Vuokraus" for that column so I'm assuming its good enough as it is. Changed in code to use lease_identifier though.